document/core/exec/instructions.rst

```diff
@@ -182,6 +182,494 @@ Where the underlying operators are non-deterministic, because they may return on
    \end{array}
 
 
+.. index:: SIMD instruction
+   pair: execution; instruction
+   single: abstract syntax; instruction
+.. _exec-instr-simd:
+
+SIMD Instructions
+~~~~~~~~~~~~~~~~~~~~
+
+SIMD instructions are defined in terms of generic numeric operators applied lane-wise based on the :ref:`shape <syntax-simd-shape>`.
```

Comment:
2. This definition of simd operations rely on the tweak to numeric operations, that lifts numeric operations to sequences of input.